flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/Projection.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.graph;
+
+import org.apache.flink.api.java.tuple.Tuple6;
+
+/**
+ * Result of projection of a connection between two vertices in a bipartite graph.
+ *
+ * @param <VK> the key type of vertices of an opposite set
+ * @param <VV> the value type of vertices of an opposite set
+ * @param <EV> the edge value type
+ */
+public class Projection<VK, VV, EV, VVC> extends Tuple6<VK, VV, EV, EV, VVC, VVC> {
```

Comment:
Missing comment for documenting `VVC`. Should `EV` be placed before `VVC`? And before `VK` and `VV`?